src/guess/100doors.js

```diff
@@ -0,0 +1,26 @@
+var doors = [];
```

Comment:
I couldn't figure out what problem this file solves even after checking the readme. When submitting prrrs, if it's not clear what a function does, maybe leave a comment?